test/suites/vm.sh

```diff
@@ -51,4 +51,13 @@ test_vm_empty() {
     # Cleanup custom changes from the default profile
     lxc profile unset default migration.stateful
   fi
+
+  echo "==> Disk mounts"
+  lxc init --vm --empty v1 -c limits.memory=128MiB -d "${SMALL_ROOT_DISK}"
+  lxc config device add v1 char disk source=/dev/zero path=/dev/zero
+  # File mounting to filesystem paths is not supported for virtual machines. This is checked at start time.
+  ! lxc start v1 || false
+  lxc config device remove v1 char
+  lxc config device add v1 char disk source=/dev/ path=/dev/
```

Comment:
you didnt try starting the VM after this line so you've not tested that directories can be added.